code/modules/food_and_drinks/food/snacks/meat.dm

```diff
@@ -87,6 +87,13 @@
 	tastes = list("meat" = 4, "scales" = 1)
 	foodtype = MEAT | RAW
 
+/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/fox
+	icon_state = "foxmeat"
+	desc = "Rawr?"
```

Comment:
It's literally a sliced up piece of a furry. There's nothing cute about this. Don't :eyes: me.